core/templates/pages/classroom-page/classroom-page.component.ts

```diff
@@ -78,6 +80,7 @@ export class ClassroomPageComponent implements OnDestroy {
       '/splash/books.svg');
 
     this.loaderService.showLoadingScreen('Loading');
+    this.diagnosticTestFeatureFlag();
```

Comment:
Function calls should start with a verb. isDiagnosticTestFeatureFlagEnabled()

Reply:
Updated